reference/tax_provider.yml

```diff
@@ -254,6 +257,9 @@ paths:
                                     name: Brutal Tax
                                     code: US
                                   id: Brutal Tax
+                            duties:
+                              - name: Brutal Duty
+                                amount: 50
```

Comment:
Another topic that is up for exploration is whether or not an item can ever have more than one custom duty associated - and, if not, whether this is likely to change in the future.

Avalara has advised that they do not support more than one duty per item.

Further exploration outside of Avalara is warranted, although also a flexible model could still be suitable even if it's only anticipated that one duty is required.